stress-ng.h

```diff
@@ -2754,6 +2772,7 @@ typedef enum {
 	OPT_sched_period,
 	OPT_sched_runtime,
 	OPT_sched_deadline,
+	OPT_sched_reclaim,
 
```

Comment:
All options need to be added to the manual too with a useful description of what that do.

Reply:
Copy, will add in v2.